[UPDATE BONUS GUIDE] Circuitbreaker w/ webinterface

blckbx opened the pull request:
### What

In this PR we update circuitbreaker to its latest version and configure the new webinterface on port 4005.

### Why

Easier configuration and nicer overview of in-flight htlcs.

#### How

Following instructions from https://github.com/lightningequipment/circuitbreaker

#### Scope

- [ ] significant change to core configuration
- [x] updated independent bonus guide
- [ ] simple bug fix

kristapsk commented on guide/bonus/lightning/circuit-breaker.md:
````diff
@@ -34,11 +34,44 @@ Table of contents
+* Configure firewall to allow incoming HTTP requests from your local network to the web server.
+
+  ```sh
+  $ sudo ufw allow 4005 comment 'allow CircuitBreaker SSL'
````
UDP is not needed here.
```suggestion
  $ sudo ufw allow 4005/tcp comment 'allow CircuitBreaker SSL'
```

kristapsk commented on guide/bonus/lightning/circuit-breaker.md:
```diff
@@ -20,7 +20,7 @@ Difficulty: Easy
 {: .label .label-green }
 
-![circuit-breaker-tweet](../../../images/circuit-breaker-tweet.png)
+![circuitbreaker-screenshot](https://raw.githubusercontent.com/lightningequipment/circuitbreaker/master/screenshot.png)
```
It would be better to keep images locally (at `../../../images/`), not hosted at GitHub. Allows to see them locally, even offline, and also will continue to work even if GitHub decides to cancel RaspiBolt.

blckbx replied:
fixed

kristapsk commented:
I see that CircuitBreaker has release tags, most recent is https://github.com/lightningequipment/circuitbreaker/releases/tag/v0.5.0. Any reason to use master instead of latest release?

blckbx commented:
> I see that CircuitBreaker has release tags, most recent is https://github.com/lightningequipment/circuitbreaker/releases/tag/v0.5.0. Any reason to use master instead of latest release?

Nope. Pushed updates:
- added tag
- added pgp key import and verification of current tag
- removed unnecessary `go build` command (`go install` is sufficient)

kristapsk commented on guide/bonus/lightning/circuit-breaker.md:
````diff
@@ -80,75 +122,35 @@ Table of contents
-   ```
-
-* Once edited, save and exit.
+To access Circuit Breaker's configuration, go to its webinterface at https://raspibolt.local:4005. Before that, please read some basics about the operating modes Circuit Breaker has to offer: [https://github.com/lightningequipment/circuitbreaker#operating-modes](https://github.com/lightningequipment/circuitbreaker#operating-modes)
````
This should go somewhere below, after Circuit Breaker is started.

kristapsk commented:
For some reason it looks installation fails for me... Any ideas?
```
$ go install
...
go: downloading github.com/remyoudompheng/bigfft v0.0.0-20200410134404-eec4a21b6bb0
# runtime/internal/sys
/usr/local/go/src/runtime/internal/sys/consts.go:13:7: StackGuardMultiplier redeclared in this block
        /usr/local/go/src/runtime/internal/sys/arch.go:27:7: other declaration of StackGuardMultiplier
/usr/local/go/src/runtime/internal/sys/consts.go:16:7: DefaultPhysPageSize redeclared in this block
        /usr/local/go/src/runtime/internal/sys/arch.go:36:7: other declaration of DefaultPhysPageSize
/usr/local/go/src/runtime/internal/sys/consts.go:20:7: PCQuantum redeclared in this block
        /usr/local/go/src/runtime/internal/sys/arch.go:40:7: other declaration of PCQuantum
/usr/local/go/src/runtime/internal/sys/consts.go:23:7: Int64Align redeclared in this block
        /usr/local/go/src/runtime/internal/sys/arch.go:43:7: other declaration of Int64Align
/usr/local/go/src/runtime/internal/sys/consts.go:30:7: MinFrameSize redeclared in this block
        /usr/local/go/src/runtime/internal/sys/arch.go:50:7: other declaration of MinFrameSize
/usr/local/go/src/runtime/internal/sys/consts.go:34:7: StackAlign redeclared in this block
        /usr/local/go/src/runtime/internal/sys/arch.go:54:7: other declaration of StackAlign
$ circuitbreaker
-bash: circuitbreaker: command not found
$ ls /home/circuitbreaker/go/bin/
ls: cannot access '/home/circuitbreaker/go/bin/': No such file or directory
$ go version
go version go1.19.3 linux/arm64
```

blckbx commented:
> For some reason it looks installation fails for me... Any ideas?

Yeah this looks like it's not compiled successfully. 
For comparison, my go version is go1.21.4 linux/amd64

M1ch43lV reviewed (approved):
No problems with installation

kristapsk commented:
> For some reason it looks installation fails for me... Any ideas?
> 
> ```
> $ go install
> ...
> go: downloading github.com/remyoudompheng/bigfft v0.0.0-20200410134404-eec4a21b6bb0
> # runtime/internal/sys
> /usr/local/go/src/runtime/internal/sys/consts.go:13:7: StackGuardMultiplier redeclared in this block
>         /usr/local/go/src/runtime/internal/sys/arch.go:27:7: other declaration of StackGuardMultiplier
> /usr/local/go/src/runtime/internal/sys/consts.go:16:7: DefaultPhysPageSize redeclared in this block
>         /usr/local/go/src/runtime/internal/sys/arch.go:36:7: other declaration of DefaultPhysPageSize
> /usr/local/go/src/runtime/internal/sys/consts.go:20:7: PCQuantum redeclared in this block
>         /usr/local/go/src/runtime/internal/sys/arch.go:40:7: other declaration of PCQuantum
> /usr/local/go/src/runtime/internal/sys/consts.go:23:7: Int64Align redeclared in this block
>         /usr/local/go/src/runtime/internal/sys/arch.go:43:7: other declaration of Int64Align
> /usr/local/go/src/runtime/internal/sys/consts.go:30:7: MinFrameSize redeclared in this block
>         /usr/local/go/src/runtime/internal/sys/arch.go:50:7: other declaration of MinFrameSize
> /usr/local/go/src/runtime/internal/sys/consts.go:34:7: StackAlign redeclared in this block
>         /usr/local/go/src/runtime/internal/sys/arch.go:54:7: other declaration of StackAlign
> $ circuitbreaker
> -bash: circuitbreaker: command not found
> $ ls /home/circuitbreaker/go/bin/
> ls: cannot access '/home/circuitbreaker/go/bin/': No such file or directory
> $ go version
> go version go1.19.3 linux/arm64
> ```

Tried `apt update` and `apt upgrade` and fetching latest circuitbreaker, still having the same issue.

But if others report no problems with installation, I'm ok with merging this.
